.eslintrc.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
+  parserOptions: {
+    parser: '@typescript-eslint/parser',
```

Comment:
Support for this is now built-in to the latest version of kcd-scripts (actually, eslint-config-kentcdodds, but that's what kcd-scripts uses).

Reply:
That's what I assumed when I checked the changelogs, but sadly enough [eslint-plugin-vue requires](https://eslint.vuejs.org/user-guide/#how-to-use-a-custom-parser) us to use `parserOptions.parser` option instead of `parser`, because the latest needs to be used internally to parse Vue files.

SFC are cool but come with their own headaches 😅